### PR TITLE
Comments: adjust outsized emoji in comments to be a smaller size

### LINKED ISSUE #15341: Reader Comments: missing and outsized emoji
### Expected behavior
All emoji load, instead of a gray box appearing, and when it loads, it shows up as the correct size.

### Actual behavior
Some emoji show up as a large gray box. Some emoji load but as large blurry photos.

### Steps to reproduce the behavior
1. Navigate to Notifications > "liked your comment" > Comment. The `twemoji` images are too big.
2. Navigate to Reader > Following > choose a post with comments > view comments. The `twemoji` explode.

### Screenshots
| First example | Second |
| --- | --- |
| IMG_5265 | IMG_5266 |

### PR DESCRIPTION
**Pull request by mindgraffiti**
Fixes #15341 
Ref. https://github.com/Automattic/p2/issues/1106 

Twemoji are sometimes used to replace Android or iOS emoji on the web. Twemoji are supported in comments. This causes an issue for iOS because the HTML for a Twemoji does not contain size information. The majority of Twemoji are also `.svg` file format, which is not supported on iOS.

This PR adds size constraints to Twemoji by looking for the CSS class `emoji`. It does not remove the gray box problem (attempted loading of an `.svg` file into an imageView). That will need fixed in Calypso https://github.com/Automattic/wp-calypso/issues/47807 or in P2.

### Alternatives considered
I did find a [list of image paths](https://github.com/wordpress-mobile/WordPress-iOS-Shared/blob/develop/WordPressShared/Core/Utility/DisplayableImageHelper.m#L106-L111) that we could filter by, but then immediately found twemoji that lived under other image paths, making this option more brittle than the route I took.

### To test
Find a comment that uses Twemoji. (Unfortunately, I don't know how to cause a twemoji replacement.)

**Test 1**
1. Navigate to Notifications > "liked your comment" > Comment. 
2. Navigate to Reader > Following > choose a post with comments > view comments. 

### Screenshots
| Before | After |
| --- | --- |
| Simulator Screen Shot - iPhone 11 Pro - 2020-12-14 at 16 25 49 | Simulator Screen Shot - iPhone 11 Pro - 2020-12-14 at 16 20 52 |
| Simulator Screen Shot - iPhone 11 Pro - 2020-12-14 at 16 26 20 | Simulator Screen Shot - iPhone 11 Pro - 2020-12-14 at 16 15 10 |

PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Review by ScoutHarris — APPROVED** [review]
I found the 'welcome David' post and viewed it's comments. 👍 looks good! Well done!

:shipit:

File: WordPress/Classes/ViewRelated/Views/WPRichText/WPRichTextFormatter.swift [hunk 1]
```diff
@@ -491,13 +491,21 @@ class AttachmentTagProcessor: HtmlTagProcessor {
...
         textAttachment.attributes = attrs
         textAttachment.html = html
 
-        let size = getSize(from: attrs)
+        var size = getSize(from: attrs)
+
+        // `twemoji` without an assigned size become outsized images
+        // https://github.com/wordpress-mobile/WordPress-iOS/issues/15341
+        if (size.height == 0 || size.height == 0) && cssClass.contains("emoji") {
+                size = CGSize(width: 20, height: 20)
```
Comment (emilylaguna) [inline WordPress/Classes/ViewRelated/Views/WPRichText/WPRichTextFormatter.swift]:
```suggestion
            size = CGSize(width: 20, height: 20)
```

**Review by emilylaguna — CHANGES_REQUESTED** [review]
✅ Confirmed this is fixed, I left 1 minor comment to fix the indenting on one of the lines.

File: WordPress/Classes/ViewRelated/Views/WPRichText/WPRichTextFormatter.swift (hunk 1 above)
Comment (mindgraffiti) [reply WordPress/Classes/ViewRelated/Views/WPRichText/WPRichTextFormatter.swift]: GH is lagging out and won't let me take the commit suggestion. Making the change manually and pushing a commit.

**Review by emilylaguna — APPROVED** [review]
LGTM! 👍

**Comment by mindgraffiti** [comment]
Thank you for the reviews @ScoutHarris @emilylaguna !

**Pull request merged**